CHANGELOG.rst

```diff
@@ -1,5 +1,41 @@
 Changelog
 =========
+1.3 (2019-09-30)
+------------------
+MLflow 1.3.0 includes several major features and improvements:
+
+Features:
+
+- The Python client now supports logging & loading models using TensorFlow 2.0 (#1872, @juntai-zheng)
+- Dramatic performance improvements when fetching runs and experiments in MLflow servers that use SQL database-backed storage (#1767, #1878, #1805 @dbczumar)
+- New ``GetExperimentByName`` REST API endpoint, callable via the Python client (#1775, @smurching)
```

Comment:
i forget, does this make `set_experiment` (and possibly other python APIs) faster?

Reply:
yep, will mention this (good idea to mention the actual user impact)